cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
The situation would be like:
"I'm done with the new feature, so I pass the cpplint and I get 18 warnings." Now my standard procedure with be to automatically fix them all, and not to guess which ones belong to my new feature because that:
1) Requires time
2) I can be mistaken and I don't include a fix that I should, or the other way around

For small fixes like this one I think it's not so important if they do not strictly belong to the PR itself since _anyway_ they will be included in the next PR.
But yes, I admit that the counterpart of this is that the reviewer has more stuff to analyse, but anyway he will have to do it sooner or later